packages/i18next/src/lib/functions.ts

```diff
@@ -198,17 +273,77 @@ export async function editLocalized<TKeys extends TFunctionKeys = string, TInter
 	target: Message,
 	options: LocalizedMessageOptions<TKeys, TInterpolationMap>
 ): Promise<Message>;
+/**
+ * Edits a deferred/replied interaction using the language `keys` from your i18next language setup.
+ * @since 2.4.0
+ * @param target The interaction to editReply.
+ * @param options The language keys to be sent.
+ * @example
+ * ```typescript
+ * // Using a string to specify the key to send
+ * await editLocalized(interaction, 'commands/ping:fail');
+ * // ➡ "Pong!"
+ * ```
+ */
+export async function editLocalized<TKeys extends TFunctionKeys = string>(
+	target: BaseCommandInteraction | MessageComponentInteraction,
+	keys: TKeys | TKeys[]
+): Promise<ReturnType<(BaseCommandInteraction | MessageComponentInteraction)['editReply']>>;
+/**
+ * Edits a deferred/replied interaction using an objects option.
+ * @since 2.4.0
+ * @param target The interaction to editReply.
+ * @param options The options to be sent, requiring at least `keys` to be passed.
```

Comment:
```suggestion
 * @param options An object of valid options, requiring at least a `keys` field.
```
I'd suggest linking to the valid options and maybe a `seealso` for the overload where the keys are described.